.github/workflows/publish.yml

```diff
@@ -22,35 +22,18 @@ jobs:
       pyi-ver: ${{ env.PYI_VER }}
       pyi-rel-ver: ${{ env.PYI_REL_VER }}
     steps:
-      - if: github.event_name == 'schedule' || github.event.inputs.workflow-ghidra-ver == 'latest'
-        name: Get Latest Ghidra Version
-        id: get_latest_ghidra_ver
-        uses: pozetroninc/github-action-get-latest-release@v0.6.0
-        with:
-          repository: NationalSecurityAgency/ghidra
-          excludes: prerelease, draft
-      - name: Set Ghidra Version from Latest
-        if: github.event_name == 'schedule' || github.event.inputs.workflow-ghidra-ver == 'latest'
-        id: format_ghidra_ver
-        run: |
-          echo "GHIDRA_VER=$(echo ${{steps.get_latest_ghidra_ver.outputs.release}} | cut -d_ -f2)" >> $GITHUB_ENV
+      - name: Set Ghidra Version to latest
+        run: echo "GHIDRA_VER=latest" >> $GITHUB_ENV
```

Comment:
Why are you setting the `GHIDRA_VER` variable in this way? It seems cleaner to do this by setting the env variable via the CI directly. It was set this way before because it was being dynamically derived (and then you need this `>> $GITHUB_ENV` pattern), but if it's fixed, it's unneeded.